include/picongpu/fields/FieldTmp.tpp

```diff
@@ -198,6 +198,26 @@ namespace picongpu
         PMACC_KERNEL(Kernel{})(mapper.getGridDim(), numWorkers)(mapper, thisBox, modifyingBox);
     }
 
+    template<uint32_t AREA, bool sumSqrt>
+    float_X FieldTmp::calculateSum()
+    {
+        constexpr uint32_t numWorkers
+            = pmacc::traits::GetNumWorkers<pmacc::math::CT::volume<SuperCellSize>::type::value>::value;
+        auto mapper = makeAreaMapper<AREA>(cellDescription);
+        FieldTmp::DataBoxType thisBox = this->fieldTmp->getDeviceBuffer().getDataBox();
+        using Kernel = SumValuesKernel<numWorkers, MappingDesc::SuperCellSize, sumSqrt>;
```

Comment:
discussed with @pordyna offline: There is no need to write an own reduce we have a global and local reduce in PMacc.
With using the already existing reduce changes in `ThreadCollective` and `FieldTmp` should be not nessesary.